app/ports/migrations/0006_add_clt_version_and_variants.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 2.1.7 on 2019-08-05 18:01
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ports', '0005_remove_models_user_osdist'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='portinstallation',
+            name='variants',
+            field=models.CharField(default='', max_length=200),
```

Comment:
I cannot really decide how much we need here, and would be tempted to either use perhaps 256 or just text with no limits. But if we use text, maybe the performance could decrease. Maybe 200 is sufficient?